packages/monaco/src/browser/style/index.css

```diff
@@ -221,3 +221,10 @@
 .monaco-editor .find-widget .matchesCount {
   font-family: var(--theia-ui-font-family);
 }
+
+.monaco-editor .peekview-widget .monaco-list-row.focused.selected .label-name,
+.monaco-editor .peekview-widget .monaco-list-row.focused.selected .label-description,
+.monaco-editor .peekview-widget .monaco-list-row.focused.selected .monaco-highlighted-label,
+.monaco-editor .peekview-widget .monaco-list-row.focused.selected .codicon {
```

Comment:
@colin-grant-work For the presence of each of the code blocks mentioned earlier, the differences are as followed in `Peek References`: 

Without the `color: var` block
![image](https://user-images.githubusercontent.com/72292738/139494265-85365e4f-6f8f-4db5-9fcd-8b56a66f5fe4.png)

Without the `color: unset` block
![image](https://user-images.githubusercontent.com/72292738/139494342-f5228d54-1d4b-4c58-8069-415535b010d4.png)

With this, it seems reasonable to get rid of the `color: var` block, but am unsure of what you intended by your comment. By the pictures, is this what you meant and is there any other place effected by this code?

Reply:
I haven't tested out any other modifications of the styling, so I'm not sure what the optimal solution is. The chief requirement that we find some combination of rules that works (i.e. produces legibly contrastive foreground and background colors) for a wide variety of themes and combinations of focus and selection state. The previous code had a problem with the combination of light theme + selection + non-focus. 

My impression as that the `monaco` package's CSS file has a bit of clutter in it: rules introduced to address particular cases that actually interfere with others. If that's the case, it might be nice to clear out some of the clutter rather than adding rules for another set of specific cases. But it may be that that's impractical, and adding some rules is the way to go. Whatever produces consistently legible text is fine with me.

Reply:
@colin-grant-work With your earlier comment about the active foreground and not active background, was your concern about the background not being active with the foreground when unselected, or was it that the foreground should not be active when unselected.
Also are you able to retest it and see what you found wrong, and if you can provide the steps to reproducing it?

Reply:
I'm unable to reproduce the mix of focused and non-focused behavior from my screenshot above. I'm not even able to find the rules that could produce such behavior, so I'm a bit confused how I ever got there.